core/templates/dev/head/pages/state_editor/StateInteractionEditorDirective.js

```diff
@@ -213,8 +215,21 @@ oppia.directive('stateInteractionEditor', [
                       UrlInterpolationService.getInteractionThumbnailImageUrl);
 
                     $scope.INTERACTION_SPECS = INTERACTION_SPECS;
-                    $scope.ALLOWED_INTERACTION_CATEGORIES = (
-                      GLOBALS.ALLOWED_INTERACTION_CATEGORIES);
+
+                    // Checks that the modal is invoked by exploration_editor or
+                    // skill_editor, topic_editor. 
+                    var currentUrl = $window.location.pathname.split('/')[1];
```

Comment:
That's not really the best naming I think. The output after the split isn't really the URL, right? It is more like the page the user is on. Might be better to name it related to that.

Reply:
Also what if the url doesn't contain `/` then this line will throw an error.

Reply:
@vojtechjelinek I feel like $window.location.path will always return something with '/'. Found this from a document: Location.pathname Is a DOMString containing an initial '/' followed by the path of the URL. https://developer.mozilla.org/en-US/docs/Web/API/Location

Reply:
@nithusha21 Done. PTAL.